npm/nameSpaceController.go

```diff
@@ -41,12 +42,12 @@ type Namespace struct {
 }
 
 // newNS constructs a new namespace object.
-func newNs(name string) (*Namespace, error) {
+func newNs(name string, exec utilexec.Interface) (*Namespace, error) {
 	ns := &Namespace{
 		name:      name,
 		LabelsMap: make(map[string]string),
 		SetMap:    make(map[string]string),
-		IpsMgr:    ipsm.NewIpsetManager(),
+		IpsMgr:    ipsm.NewIpsetManager(exec),
```

Comment:
As far as I understood our code, there is no reason to tie IpsMgr and IptMgr in Namespace struct. In my opinion, it is better to locate them to other places (e.g., IpsMgr in npMgr, IptMgr in networkpolicyController). So, it will be easier to use our main code and UTs. I felt like there are multiple places to pass exec, etc now. However, I think that is beyond of this PR. We will fix it in other PRs. Any thoughts?